tasks/main.yml

```diff
@@ -38,21 +38,35 @@
     mode: 0644
   notify: restart youtrack service
 
-- name: download youtrack jar file
+- name: download youtrack zip file
   ansible.builtin.get_url:
-    url: "https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.jar"
-    dest: "{{ youtrack_releases_dir }}/youtrack-{{ youtrack_version }}.jar"
-    checksum: "sha256:https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.jar.sha256"
+    url: "https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.zip"
+    dest: "{{ youtrack_releases_dir }}/youtrack-{{ youtrack_version }}.zip"
+#    checksum: "sha256:https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.zip.sha256"
```

Comment:
why this was commented out?

Reply:
Is this reproducible? Could you please provide error message?

Reply:
Yeap. this is reproducible. 

Text of error message: 

TASK [ary2 : download youtrack zip file] *********************************************************************************************************************************************************************
fatal: [u2]: FAILED! => {"changed": false, "msg": "Unable to find a checksum for file 'youtrack-2023.1.17582.zip' in 'https://download.jetbrains.com/charisma/youtrack-2023.1.17582.zip.sha256'"}

Meanwhile i check link from official site: https://download.jetbrains.com/charisma/youtrack-2023.1.17582.zip.sha256

Both link indentical.

Reply:
Both links work for me:
![image](https://github.com/oukooveu/ansible-role-youtrack/assets/2072275/18ca4a89-f48b-4124-81e1-e18523db9869)

Reply:
For me too, but while ansible trying to complete this step that's crushing every times.

Reply:
The playbook below works as expected, please check your setup:
```- name: test download
  hosts: localhost
  gather_facts: false
  vars:
    youtrack_version: 2023.1.17582
    youtrack_releases_dir: /tmp
  tasks:
    - name: download youtrack zip file
      ansible.builtin.get_url:
        url: "https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.zip"
        dest: "{{ youtrack_releases_dir }}/youtrack-{{ youtrack_version }}.zip"
        checksum: "sha256:https://download.jetbrains.com/charisma/youtrack-{{ youtrack_version }}.zip.sha256"
```